test/test_distributions.py

```diff
@@ -15,6 +17,45 @@
     TEST_NUMPY = False
 
 
+# Register all distributions for generic tests.
+Example = namedtuple('Example', ['Dist', 'params'])
```

Comment:
Looks good. Is the plan to use this mainly for shape testing? We may need to extend the fixture to support other tests (in later PRs).

Reply:
Yes, we can add more fields as we use this fixture for more tests (beyond just shape).

Reply:
BTW I have a slight ideological preference for this data-only approach to parameterized tests. I find it improves transparency to limit parameterized test data to plain old data, and keep the logic that manipulates that data in `TestCase` classes or other functions. I'd like to hear your opinions about test fixture style.

Reply:
I don't really have too many opinions. I find it nice for fixtures to contain the bare minimum specification needed to run the tests - i.e. just pure data, and have test methods themselves or other helper utilities manipulate that further as the case may be. 

If there is a lot of setup needed to construct the fixture, then it could be abstracted into its own class or function - for instance constructing a JSON request for a service. If all methods need to do the same manipulation, it is again nice to abstract the fixture setup into a separate class or function. If each test manipulates the data somewhat differently, then the test methods itself should consume the raw fixture data. Once these fixtures start containing multiple fields (expected test values, etc.) it may get unwieldy and it may make sense to have separate fixtures for different tests rather than have a mega fixture.